### PR TITLE
Check annotations labels and rename them if needed

### PR DESCRIPTION
**Pull request by ArthurNguyen**
This is to check and avoid annotations being ignored because they were mislabelled

(usual suspects are 
- bad?
- bads
- bads_
or some combinations of those, instead of 
- bad_

### CODE REVIEW

File: scripts/tasks/checks/logs_labels_2023-10-19__16-00.txt [hunk 1]
```diff
@@ -0,0 +1,205 @@
+2023-10-19__16-00
+
+Files to amend
+
+Files with no bad_
+sub-P05-G2_task-oddball_run-2
+sub-P15-G4_task-oddball_run-2
+sub-P46-G3_task-oddball_run-1
```
Comment (mscheltienne) [inline scripts/tasks/checks/logs_labels_2023-10-19__16-00.txt:8]: Should that log be committed?

Comment (ArthurNguyen) [reply scripts/tasks/checks/logs_labels_2023-10-19__16-00.txt:8]: unsure, I'm not sure where I should save a log of what has been checked so far and does not need to be rechecked?

**Comment by mscheltienne** [comment]
I updated the notebook, untested and probably broken, but cleaner. I let you modify it further and fix the bugs I might have added.

File: scripts/tasks/checks/logs_labels_2023-10-19__16-00.txt (hunk 1 above)
Comment (mscheltienne) [reply scripts/tasks/checks/logs_labels_2023-10-19__16-00.txt:8]: Personally, I would not bother looking through the log and would simplfy re-run the script. You are not preloading the data in memory, so the I/O operations should be fast enough.

Comment (ArthurNguyen) [reply scripts/tasks/checks/logs_labels_2023-10-19__16-00.txt:8]: still quite slow! probably specific to our storage though... but I agree, it's worth redoing to be safe

Comment (mscheltienne) [reply scripts/tasks/checks/logs_labels_2023-10-19__16-00.txt:8]: > probably specific to our storage though

I think so.. should not take more than 1 or 2 minutes I think.

**Review by mscheltienne — APPROVED** [review]
We could probably write a purely automatic function which checks for label names pattern and apply it automatically at the end of step3.
But hey, good enough is good enough now :)

Also leaving the log in, in case you need it. But I don't think it will be useful.

**Pull request merged**